### ISSUE TITLE
Filter value inserted multiple times (array field)

### ISSUE THREAD

**Issue by gsig91** [indice-co/Indice.Angular#42]
Greetings,

We are using the ```lib-list-view``` inside of a ```lib-view-layout``` and we are got a questiong about the usage of filters.

We noticed that we can insert multiple times the same filter.

 What we got so far:
 
```typescript
const checkpointTypeSearchOption: SearchOption = {
    field: 'checkpointTypeCodes',
    name: 'ΤΡΕΧΟΝ ΣΗΜΕΙΟ ΕΛΕΓΧΟΥ',
    dataType: 'array',
    options: [
        {
            "value": "Value1",
             "label": "Submit value 1"
        },
        {
            "value": "Value2",
            "label": "Submit value 2"
        }
    ],
    multiTerm: true
}


```
If I search with `Value1` then `Value2` then again `Value1`  the produced filter looks like this 
```url
&filter=caseTypeCodes::eq::Value1,caseTypeCodes::eq::Value2,caseTypeCodes::eq::Value1
```

Is this the desired behaviour?

If it is, is there a way/event that we can use to not permit the insertion of the same filter value multiple times?

**Comment by Pt4r** [comment]
@ChristosAsvestopoulos could you check this?

**Comment by ChristosAsvestopoulos** [comment]
@gsig91 the aforementioned issue has been confirmed and its resolution is in progress by our development team.
We will notify you once the issue is resolved.

**Comment by ChristosAsvestopoulos** [comment]
@gsig91 we would like to inform you that the issue has been resolved in version: 0.2.167-beta.34 of @indice/ng-components.
Kind Regards!
